bin/update_go

```diff
@@ -0,0 +1,150 @@
+#!/bin/sh
+
+set -e
+
+. ../../pkg/logger-sh/logger.sh
+. ../../pkg/stdlib-sh/stdlib.sh
+
+usage() {
+    echo "usage: update_go [onrsh]
+        -o    version     
+        -n    version
+        -r    repository (rancher/image-build-base)
+        -s    display the hardened-build-base version of a target repository
+        -h    show help
+
+    examples: 
+        update_go -o 1.16.3b7 -n 1.17.3b7 -r image-build-etcd"
+}
+
+get_repo() {
+    repo=$1
+    repo_path="${HOME}/go/src/github.com/rancher/${repo}"
```

Comment:
can you make this path customizable, I for one have my GOPATH in different path and therefor the rke2 and k3s repos are not in that path

Reply:
What would you suggest in this case?

Reply:
Good call, will do.

Reply:
We couldt look to $GOPATH env var and if it doesn't exist assume $HOME/go/src/github.com/rancher/. We could also include a flag to override the location.

Reply:
Adding a `-p` would probably be easiest. If it's given and given with an arg, it overrides the default.